notebooks/MAST/downloading_transiting_planet_data/downloading_transiting_planet_data.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<p>Even though I like the idea of teaching functions, having this as a function messes with the story telling aspect typically associated with a tutorial. I think each of these steps should be walked through one-by-one. </p><p> </p><p>If you want to combine them into a function later, I'm ok with that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>